docs/DeveloperGuide.md

```diff
@@ -638,339 +644,183 @@ Use case ends.
     * 2a2. User enters the command again.
     * Steps 2a1-2a2 are repeated until the data entered are correct.
     * Use case resumes from step 3.
-* 2b. HealthContact detects that the patient already exists in the database.
-    * 2b1. HealthContact shows an error message.
-    * 2b2. User enters the command again.
-    * Steps 2b1-2b2 are repeated until the patient does not exist in the database.
-    * Use case resumes from step 3.
+* 8a. HealthContact detects an error in the format of the command entered.
+    * 8a1. HealthContact shows an error message.
+    * 8a2. User enters the command again.
+    * Steps 8a1-8a2 are repeated until the data entered are correct.
+    * Use case resumes from step 9.
 
-**Use case: Adding an appointment**
+**Use case**: UC-02 - An existing patient's details are outdated<br>
 
 **MSS**
-
 1. Patient calls the clinic to make an appointment.
-2. User enters add command with the detailed information of appointment to be added.
-3. HealthContact adds the appointment
-
-    Use case ends.
+2. User uses the find patient command to find the patient's details by name.
+3. HealthContact displays the patient on the patient list.
+4. User selects the patient to view his/her details. 
+5. User asks the patient for his/her updated personal details. 
+6. Patient provides his/her updated personal details to the user. 
+7. User enters the edit patient command with the updated patient details. 
+8. HealthContact edits the patient and displays the updated patient details. 
+9. Patient provides the appointment information. 
```

Comment:
Do you think this use case does not need add appointment? Because a patient can j call to let the clinic know that their phone number or email address has changed? Say to send their bills or something

Reply:
but usually thats not the case, usually only when they visit the clinic then the details are updated